split.py

```diff
@@ -9,15 +12,21 @@
 # Load processed data
 all_spectro = utils.load_dump(config.EMOTIFY_SPECTROGRAM_DUMP_PATH)
 
-labels = utils.load_labels(config.EMOTIFY_LABELS_DUMP_PATH)
```

Comment:
If you don't need `utils.load_labels` anymore, could you remove the function ? Unless it is used somewhere else.

Reply:
Don’t think it is really necessary to remove it right now, it could become useful.